updateUi.py

```diff
@@ -0,0 +1,92 @@
+from tkinter import *
+from GameData import GameData
+from State import State
+from StatEnum import Stat
+from CharacterInstance import CharacterInstance
+from UserLogs import UserLogs
+
+
+class updateUi(Frame):
+    def __init__(self, user_logs, master=None):
+        Frame.__init__(self, master)
+        self.charattribute = []
+        self.createwidgets()
+        self.user_logs = user_logs
+
+    def update_f(self):
```

Comment:
"""
update_f updates a logged character with a new state given by the user
"""